src/js/utils.js

```diff
@@ -252,4 +252,16 @@ export function getChromeBrowserVersion(){
     } else {
         return -1;
     }
-}
\ No newline at end of file
+}
+
+export function isCitrixWebRTCSupported() {
```

Comment:
why is this part of common utils class? Can this be made part of CitrixStrategy?

Reply:
Moved